processor/processor.go

```diff
@@ -473,6 +476,8 @@ func loadConfig() {
 	config.RegisterIntConfigVariable(0, &pipelineBufferedItems, false, 1, "Processor.pipelineBufferedItems")
 	config.RegisterIntConfigVariable(2000, &subJobSize, false, 1, "Processor.subJobSize")
 	config.RegisterDurationConfigVariable(time.Duration(5000), &maxLoopSleep, true, time.Millisecond, []string{"Processor.maxLoopSleep", "Processor.maxLoopSleepInMS"}...)
+	config.RegisterDurationConfigVariable(5*time.Minute, &storeTimeout, true, time.Millisecond, "Processor.storeTimeout")
```

Comment:
Default value passed needs correction.

Reply:
5 min = time.Duration(5*time.Minute/time.Millisecond)

Reply:
Oh got it, I will fix it. 

I find the base unit confusing (4th param). I am expecting to get what I provide as a default.